### PR TITLE
baremetal: remove filters for sig-network and any disconnected tests

### PR DESCRIPTION
**Pull request by stbenjam**
This PR removes the filters for any tests that belong to sig-network or
do not work on disconnected. Some tests will be skipped in code in
openshift-tests, as there's now logic to filter disconnected tests out.

### CODE REVIEW

**Comment by stbenjam** [comment]
Waiting for https://github.com/openshift/origin/pull/26019 to land

File: ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh [hunk 1]
```diff
@@ -47,12 +47,23 @@ packet)
     # shellcheck source=/dev/null
     source "${SHARED_DIR}/ds-vars.conf"
     copy_test_binaries
-    export TEST_PROVIDER=\"\"
+
+    # Currently all v6 deployments are disconnected, so we have to tell
+    # openshift-tests to exclude those tests that require internet
+    # access.
+    if [[ "$IP_STACK" -eq "v6" ]];
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh:54]: I don't think `IP_STACK` it's available in [`ds-vars.conf`](https://github.com/openshift/release/blob/fa63247905df5522e3271f5f7d6d39aea9e6a65d/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L102-L104), it must be added previously there in the `devscript-setup` step

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh]: Oh thanks, good catch

**Comment by stbenjam** [comment]
Just waiting on https://github.com/openshift/origin/pull/26019, and I'll take this out of draft to see how it does on rehearse.

**Comment by stbenjam** [comment]
These tests still fail with

```
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should allow egress access to server in CIDR block
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should allow ingress access from updated namespace
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should allow ingress access from updated pod
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should deny ingress access to updated pod
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce egress policy allowing traffic to a server in a different namespace based on PodSelector and NamespaceSelectoE
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce except clause while egress access to server in CIDR block
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce multiple egress policies with egress allow-all policy taking precedence
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce multiple ingress policies with ingress allow-all policy taking precedence
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce multiple, stacked policies with overlapping podSelectors
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policies to check ingress and egress policies can be controlled independently based on PodSelector
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy based on NamespaceSelector with MatchExpressions
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy based on PodSelector and NamespaceSelector
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy based on PodSelector or NamespaceSelector
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy based on PodSelector with MatchExpressions
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy based on Ports
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy to allow traffic from pods within server namespace based on PodSelector
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy to allow traffic only from a different namespace, based on NamespaceSelector
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should enforce updated policy
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should ensure an IP overlapping both IPBlock.CIDR and IPBlock.Except is allowed
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should not allow access by TCP when a policy specifies only SCTP
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should stop enforcing policies after they are deleted
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should support a 'default-deny-all' policy
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should support a 'default-deny-ingress' policy
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should support allow-all policy
      \[sig-network\] NetworkPolicy \[LinuxOnly\] NetworkPolicy between server and client should work with Ingress,Egress specified together
```

This one needs to be added to the disconnected list (https://github.com/openshift/origin/pull/26050)
`[sig-network\] Networking should provide Internet connection for containers`

**Comment by stbenjam** [comment]
IPv6 pass: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17445/rehearse-17445-pull-ci-openshift-installer-master-e2e-metal-ipi-ovn-ipv6-trigger/1379808267013722112

Upgrade pass: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17445/rehearse-17445-periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-upgrade/1379808270104924160

IPv4 pass: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17445/rehearse-17445-periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-virtualmedia/1379572281990189056

Dualstack seems to be having issues with sig-builds, not sure why. :-\

**Comment by stbenjam** [comment]
These are failing on dualstack:

```
+      \[sig-builds\]\[Feature:Builds\] clone repository using git:// protocol  should clone using git:// if no proxy is configured
+      \[sig-builds\]\[Feature:Builds\] custom build with buildah  being created from new-build should complete build with custom builder image
+      \[sig-builds\]\[Feature:Builds\] imagechangetriggers  imagechangetriggers should trigger builds of all types
+      \[sig-builds\]\[Feature:Builds\] oc new-app  should fail with a --name longer than 58 characters
+      \[sig-builds\]\[Feature:Builds\] oc new-app  should succeed with a --name of 58 characters
+      \[sig-builds\]\[Feature:Builds\] oc new-app  should succeed with an imagestream
+      \[sig-builds\]\[Feature:Builds\] prune builds based on settings in the buildconfig  buildconfigs should have a default history limit set when created via the group api
+      \[sig-builds\]\[Feature:Builds\] prune builds based on settings in the buildconfig  should prune builds after a buildConfig change
+      \[sig-builds\]\[Feature:Builds\] prune builds based on settings in the buildconfig  should prune canceled builds based on the failedBuildsHistoryLimit setting
+      \[sig-builds\]\[Feature:Builds\] prune builds based on settings in the buildconfig  should prune completed builds based on the successfulBuildsHistoryLimit setting
+      \[sig-builds\]\[Feature:Builds\] prune builds based on settings in the buildconfig  should prune errored builds based on the failedBuildsHistoryLimit setting
+      \[sig-builds\]\[Feature:Builds\] prune builds based on settings in the buildconfig  should prune failed builds based on the failedBuildsHistoryLimit setting
+      \[sig-builds\]\[Feature:Builds\]\[valueFrom\] process valueFrom in build strategy environment variables  should fail resolving unresolvable valueFrom in docker build environment variable references
+      \[sig-builds\]\[Feature:Builds\]\[valueFrom\] process valueFrom in build strategy environment variables  should fail resolving unresolvable valueFrom in sti build environment variable references
+      \[sig-builds\]\[Feature:Builds\]\[valueFrom\] process valueFrom in build strategy environment variables  should successfully resolve valueFrom in docker build environment variables
+      \[sig-builds\]\[Feature:Builds\]\[valueFrom\] process valueFrom in build strategy environment variables  should successfully resolve valueFrom in s2i build environment variables
+      \[sig-imageregistry\]\[Feature:ImageTriggers\] Image change build triggers TestSimpleImageChangeBuildTriggerFromImageStreamTagSTI
```

Going to re-instate them while we investigate them as well.

**Comment by stbenjam** [comment]
We've seen this PR pass the variations of metal at least once: https://github.com/openshift/release/pull/17445#issuecomment-815100916. We also see it pass locally in our dev environments.

It's hard to get pj-rehearse to pick the exact jobs we want, so we've agreed to land this, and I'll watch the release dashboard carefully today just in case we missed something.

**Pull request merged**